### ISSUE TITLE
Performance problems on 0.27 comapred to 0.24

### ISSUE THREAD

**Issue by bob-at-author-it** [goto100/xpath#87]
We are seeing a marked increase in the length of time queries are taking (much longer) and have pinned xpath back to 0.24. Is this a known issue?

**Comment by JLRishe** [comment]
Yes, this is a known issue (#78). If you are seeing a significant degradation, I would advise sticking with 0.24 until the issue can be resolved.

**Comment by bob-at-author-it** [comment]
We have done that, thanks. Is this issue being investigated  currently?

**Comment by JLRishe** [comment]
Not at the moment. I definitely want to fix it, but that I'm just one person and have limited time, I'm not sure when I'll be able to get to it.

**Comment by fpmk** [comment]
any news? still can't move to 29 version because of performance issues.

**Comment by JLRishe** [comment]
@bob-at-author-it @fpmk 

I'm sorry that this has gone untouched for so long. As I said previously, I have limited time to look into this, but I am going to try to spend some looking at this this month.

**Comment by fpmk** [comment]
Thanks man. Seems all good.
